pivot.coffee

```diff
@@ -409,6 +410,13 @@ callWithJQuery ($) ->
                 len++
             return len
 
+        #cell click callback generator
+        makeClickCallback = (cb,cellValue,attributes,keys,data) ->
+            filters = {}
+            for own key,attr of attributes
+                filters[attr] = keys[key]
+            return (e) -> cb(e,cellValue,filters,data)
```

Comment:
Hi, sorry for the late answer.
Indeed the best would be to be able to return all matching records for the selected cell, but since this isn't already in memory, it might have really bloated the footprint. I felt it wasn't worth it especially considering that clicking a cell might be a pretty rare use case after all.
So I'm just returning the filters here to let the caller rebuild the data at call time - which isn't all that hard.